app/javascript/components/public_health/AdvancedFilter.js

```diff
@@ -137,6 +137,12 @@ class AdvancedFilter extends React.Component {
           type: 'option',
           options: ['Morning', 'Afternoon', 'Evening', ''],
         },
+        {
+          name: 'manual-contact-attempts',
+          title: 'Manual Contact Attempts (Number)',
+          description: 'All records with certain number of manual contact attempts',
```

Comment:
nothing wrong with this, but was wondering if we wanted to reach out to PO team for exact language for this?

Reply:
confirming now...

Reply:
updated